src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/Bjorhus.cpp

```diff
@@ -227,25 +226,33 @@ std::optional<std::string> ConstraintPreservingBjorhus<Dim>::dg_time_derivative(
       char_speeds.at(a) -= negative_lambda0;
     }
   }
+  // If the chosen boundary lies inside the apparent horizon, return here
+  if (helpers::min_characteristic_speed(char_speeds) >= 0.) {
```

Comment:
when do we expect to be applying the BC at the horizon? I'm a bit surprised that you need to handle this case since we can now specify different boundary conditions (including `Outflow`) on different boundaries

Reply:
You're right in part and its really the help text that has to be corrected (which I have done now). The exception case is when we have a contracting outer boundary and the net incoming char speed for, say, `VMinus` becomes 0. This may not be a realistic case, but that is what this block is supposed to handle. And lets see what it is doing anyway: its checking if all speeds are positive, skip further computation. Even if we remove this box entirely, the same behavior will ensue later on, with the only difference being we'll calculate a lot of things and never use them.

Reply:
okay, that sounds good! I think the comment is clear now :) Thanks!